### PR TITLE
[feat18]: Changes as per production

### PR DESCRIPTION
**Pull request by Abhishek-Jain-1925**
[feat18]: Changes as per production

Workdone are as follows - 
1. Logout func after employee completed profile.
2. Add updated tooltips to resp fields
3. Update confirm messages
4. minimum exp. in months should be 1

### CODE REVIEW

File: src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -23,6 +23,8 @@ const Navbar = () => {
           await logoutService();
           dispatch(logout());
           window.localStorage.removeItem("token");
```
Comment (Payal13) [inline src/components/Navbar/index.jsx]: move this to a common method and use it wherever needed

File: src/components/Resume/index.js [hunk 2]
```diff
@@ -358,6 +362,11 @@ const Resume = forwardRef(({ data }, ref) => {
             const response = await completeProfileService({
               profile_id: profile?.id,
             });
+            await logoutService();
+            dispatch(logout());
+            window.localStorage.removeItem("token");
```
Comment (Payal13) [inline src/components/Resume/index.js]: use here

**Review by Payal13 — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.jsx (hunk 1 above)
Comment (Payal13) [reply src/components/Navbar/index.jsx]: otherwise use directly localstorage.clear() to optimize performace

Comment (Abhishek-Jain-1925) [reply src/components/Navbar/index.jsx]: Done!

File: src/components/Resume/index.js (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/components/Resume/index.js]: Done!

File: src/components/Navbar/index.jsx (hunk 1 above)
Comment (Abhishek-Jain-1925) [reply src/components/Navbar/index.jsx]: Done!

File: src/components/Resume/index.js (hunk 2 above)
Comment (Abhishek-Jain-1925) [reply src/components/Resume/index.js]: Done!

**Review by Payal13 — APPROVED** [review]

**Pull request merged**